analyse-contributions/src/types.ts

```diff
@@ -31,7 +34,9 @@ export type Issue = Timestamps & {
   number: number;
   title: string;
   comments_url: string;
+  comments: number;
   html_url: string;
+  state_reason;
```

Comment:
This can be `completed`, `not_planned`, `reopened` or null. According to https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#update-an-issue

Reply:
Done